drivers/applications/tla2528_input_pin_demo.cpp

```diff
@@ -0,0 +1,47 @@
+#include "../hardware_map.hpp"
+#include <array>
+#include <libhal-util/bit.hpp>
+#include <libhal-util/i2c.hpp>
+#include <libhal-util/serial.hpp>
+#include <libhal-util/steady_clock.hpp>
+#include <libhal/input_pin.hpp>
+#include <libhal/units.hpp>
+#include <tla2528.hpp>
+#include <tla2528_adapters.hpp>
+
+using namespace hal::literals;
+using namespace std::chrono_literals;
+
+namespace sjsu::drivers {
+
+void application(application_framework& p_framework)
+{
+
+  hal::byte i2c_address = 0x10;  // 0x10 is the tla address for no resistors
+
+  auto& terminal = *p_framework.terminal;
+  auto& i2c = *p_framework.i2c;
+  auto& steady_clock = *p_framework.steady_clock;
```

Comment:
```suggestion
  auto& terminal = *p_framework.terminal.value();
  auto& i2c = *p_framework.i2c.value();
  auto& steady_clock = *p_framework.steady_clock.value();
```

Same here

Reply:
I'm marking this as resolved since the one above with the same topic of options is marked as resolved.